src/features/Builder/ActivityItems/ItemConfiguration/ItemSettingsController/ItemSettingsController.tsx

```diff
@@ -0,0 +1,91 @@
+import { ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Controller, FieldValues } from 'react-hook-form';
+import { FormControl, FormLabel, FormGroup, FormControlLabel, Checkbox } from '@mui/material';
```

Comment:
Maybe it's better to use the existent CheckboxController from the FormComponents?